### PR TITLE
Th update numbers script added (and updated numbers)

### PR DESCRIPTION
**Pull request by virtue3**
* Added convert_nanp.js - this is to be run locally to update the project after pulling down the appropriate csv file from the NANPA site.
* Added testing protocol for script to verify that it isn't breaking anything unexecptedly
* Added rules to ignore some numbers that were removed / mislabled. (900, 911, etc)
* Made this pretty simple for the end user to update the package and push again.  Which we should be doing fairly more often.

### CODE REVIEW

File: e164-world.json
Comment (adamjanower) [inline e164-world.json:1]: Why do we need these?

**Review by adamjanower — COMMENTED** [review]
This looks right to me but most of my context is just around updating in both places, so I'd defer to others for approval.

File: e164-world.json
Comment (virtue3) [reply e164-world.json:1]: they're in the original data set, and not in the north american data set coming from NANP. we don't actually need them, but would rather keep it the same than change it.

File: convert_napn.js
Comment (d-natoli) [inline convert_napn.js]:
```suggestion
// these are the country strings we expect for countries in the NAPN dataset
```

File: e164-world.json
Comment (d-natoli) [reply e164-world.json:1]: Is this file generated? If so should we add a comment at the top when it's generated to say don't edit directly?

File: e164.js
Comment (d-natoli) [inline e164.js:139]: There are a lot of ones removed, like this one. Is that an issue? Will it cause errors?

File: e164.js
Comment (d-natoli) [inline e164.js:32]: Is this file also generated by your script? If so should we add a comment?

**Review by d-natoli — APPROVED** [review]
Hard to know if this is fine, I mean it looks fine but 🤷‍♂. Added some questions that if you are unsure, you should find the answer to before merging.

File: e164-world.json
Comment (virtue3) [reply e164-world.json:1]: It's not, I split it out manually from the original data.  Not sure what to add as a comment.

File: e164.js
Comment (virtue3) [reply e164.js:139]: I specifically commented about this in the tester file.  This number is actually Canadian.  And was replaced properly in the new dataset.

File: e164.js
Comment (virtue3) [reply e164.js:32]: Only a part of the file is generated, Ill add a comment.  I regex into the data structure and modify it inline.

File: e164-world.json
Comment (virtue3) [reply e164-world.json:1]: also no comments in json, whoops.

**Comment by virtue3** [comment]
@d-natoli this is -proven- to be fine via the tests (provided the tests are correct, thats another problem).  This should be identical to the original ruby changes I published, just in JS now and build directly from the csv rather than hand massaged.

**Pull request merged**
